src/Cohesity.Powershell/Scripts/Utility/Set-CohesityCmdletConfig.ps1

```diff
@@ -1,11 +1,12 @@
 # the configuration structure for cohesity cmdlet
 class CohesityConfig {
     # the version will be helpful while upgrading the configuration
-    [int]$Version = 1
+    [int]$Version = 2
```

Comment:
What version is this?

Reply:
This version is useful in upgrading the cmdlet configurations. For example someone at version '1' and wants to upgrade to version 'n'. The upgrade process will start from '1', '2', '3'................'n-2', 'n-1', 'n'.

Reply:
this should be tied to powershell version ?

Reply:
Removed the version, as it has no relevance. The cmdlet upgrade module will find the missing attributes and add them.